js/client/modules/@arangodb/testsuites/recovery_cluster.js

```diff
@@ -249,34 +249,36 @@ function recovery (options) {
       status: false
     };
   }
-  options.cluster = true;
+  let localOptions = _.clone(options);
+  localOptions.cluster = true;
+  localOptions.enableAliveMonitor = false;
```

Comment:
We don't wan't to track pids here, do we?

Reply:
yes. the test execution crashes servers from within server testpoints, and we can't properly track / disable that.